client/cmd/internal/plan/plan.go

```diff
@@ -0,0 +1,35 @@
+package plan
+
+type Plan struct {
+	ProjectName string                               `json:"project_name"`
+	Job         OperationByNamespaces[*JobPlan]      `json:"job"`
+	Resource    OperationByNamespaces[*ResourcePlan] `json:"resource"`
+}
+
+func (p Plan) GetResult() Plan {
+	return Plan{
+		ProjectName: p.ProjectName,
+		Job:         p.Job.getResult(),
+		Resource:    p.Resource.getResult(),
+	}
+}
+
+func (p Plan) Merge(otherPlan Plan) Plan {
```

Comment:
Can you please help explain this operation.

Reply:
This operation will merge between jobs and between resources plan. This will help to merge multiple plan file into one before being processed by apply operation